Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -1463,7 +1463,8 @@ static EditorEventListener()
         /// Used to register newly created instances in edit mode.
         /// Initially handled by using ExecuteAlways, but this attribute causes the instance to be destroyed as we enter play mode, which is disruptive to services.
         /// </summary>
-        private void OnValidate()
+        [UnityEditor.Callbacks.DidReloadScripts] // This is workaround for a known unity issue when calling refresh assetdatabase from inside a on validate scope.
```

Comment:
Also wonder if we can just stop calling `AssetDatabase.Refresh()` in `ProjectPreferences`, which seems to be the root of this issue. There's a comment saying 

>// Sometimes Unity has weird bug where asset file exists but Unity will not load it resulting in _instance = null. 
                        // Force refresh of asset database before we try to access our preferences file

but I haven't been able to repro

Reply:
I also couldn't repro this one.